.github/workflows/branch_sync.yml

```diff
@@ -0,0 +1,103 @@
+name: Sync PR
+
+on:
+  schedule:
+    - cron: '33 15 * * 1-5'  # 15:33 UTC Mon-Fri
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: Branch to merge into master
+        required: false
+  # push:
+  #   branches:
+  #     - master
+  #     - '8.*.x'
```

Comment:
```suggestion
  push:
    branches:
      - master
  #     - '8.*.x'
```

Reply:
Done (just without master as that's not needed)